### DISCUSSION TITLE
Why does it take weeks for an extension to be merged?

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#673] [Q&A] [answered]
^

I want to know why it takes so long. :(

**Comment by yuri-kiss** [comment]
garbo has the manually review each extension

**Comment by ghost** [comment]
> garbo has the manually review each extension

Will there ever be new reviewers or smth?

**Comment by yuri-kiss** [comment]
i think there is lol but have not seen many actually do it

**Comment by NexusKitten** [comment, marked as answer]
Like @SurvExe1Pc mentioned, Garbo is the only one that can merge extensions, which requires rigorous bug testing and safety checks. He also works a job on the weekdays, so the weekends are the only time Garbo has time to work on it. While we don't have any other reviewers other than Garbo, anybody can help by doing their own bug testing on any PR.
